docker-compose.yaml

```diff
@@ -18,16 +25,18 @@ services:
 
   bump:
     <<: *devbox
+    <<: *mount-app-and-user-git-config
     entrypoint: docker/bump_version.sh
-    volumes:
-      - ./:/app
-      - ~/.gitconfig:/home/columbo/.gitconfig # allow script to commit as user
 
   # generate and serve the project documentation locally
-  mkdocs:
-    image: "squidfunk/mkdocs-material:6.0.2"
-    volumes:
-      - ./:/docs
+  mkdocs: &mkdocs
+    <<: *devbox
+    entrypoint: "mkdocs"
     ports:
       - "8000:8000"
+    command: ["serve", "--dev-addr=0.0.0.0:8000"]
 
+  mike:
```

Comment:
I dont understand the `mike` entrypoint. If this is intentional, thats fine! just not sure if you were testing something here

Reply:
ah, nevermind - I checked out pypi and the package makes sense